tests/t220-elemrestriction.c

```diff
@@ -0,0 +1,55 @@
+/// @file
+/// Test creation, use, and destruction of an element restriction oriented
+/// \test Test creation, use, and destruction of an element restriction oriented
+#include <ceed.h>
+
+int main(int argc, char **argv) {
+  Ceed ceed;
+  CeedVector x, y;
+  CeedInt num_elem = 6, P = 2, dim = 1;
+  CeedInt ind[P*num_elem];
+  bool orient[P*num_elem];
+  CeedScalar a[num_elem+1];
+  const CeedScalar *yy;
+  CeedElemRestriction r;
+
+  CeedInit(argv[1], &ceed);
+
+  CeedVectorCreate(ceed, num_elem+1, &x);
+  for (CeedInt i=0; i<num_elem+1; i++)
+    a[i] = 10 + i;
+  CeedVectorSetArray(x, CEED_MEM_HOST, CEED_USE_POINTER, a);
+
+  for (CeedInt i=0; i<num_elem; i++) {
+    ind[2*i+0] = i;
+    ind[2*i+1] = i+1;
+    // flip the dofs on element 1,3,...
+    orient[2*i+0] = (i%(2))*-1 < 0;
+    orient[2*i+1] = (i%(2))*-1 < 0;
+  }
+  CeedElemRestrictionCreateOriented(ceed, num_elem, P, dim, 1,
+                                    num_elem+1, CEED_MEM_HOST,
+                                    CEED_USE_POINTER, ind, orient, &r);
+  CeedVectorCreate(ceed, num_elem*2, &y);
+  CeedVectorSetValue(y, 0); // Allocates array
```

Comment:
This line shouldn't be needed in this case.

Reply:
That's probably in a few other tests too but it shouldn't be. It's a very early (4-5 years ago?) holdover

Reply:
We set `y` to zero in t200 as well, should I delete that as well?

Reply:
Ok, I'll check and update them as well.